--- a/usr/lib/qt5/qml/Sailfish/Silica/Button.qml	2017-10-28 18:02:28.915823414 +0200
+++ b/usr/lib/qt5/qml/Sailfish/Silica/Button.qml	2017-10-28 18:08:45.281688926 +0200
@@ -35,6 +35,7 @@
 import QtQuick 2.0
 import Sailfish.Silica 1.0
 import Sailfish.Silica.private 1.0
+import QtFeedback 5.0

 MouseArea {
     id: button
@@ -57,7 +58,19 @@
         button.DragFilter.end()
         pressTimer.stop()
     }
-    onPressed: button.DragFilter.begin(mouse.x, mouse.y)
+
+    ThemeEffect {
+        id: buttonPressEffect
+        effect: ThemeEffect.PressWeak
+    }
+
+    onPressed: {
+        button.DragFilter.begin(mouse.x, mouse.y)
+        buttonPressEffect.play()
+    }
+
+    onClicked: buttonPressEffect.play()
+
     onPreventStealingChanged: if (preventStealing) button.DragFilter.end()

     height: Theme.itemSizeExtraSmall
